### ISSUE TITLE
Searching for BPMN yields 0 results - but they are there?

### ISSUE THREAD

**Issue by hegelstad** [iconify/icon-sets#85]
Hi team,

I was surprised to find that I can find the bpmn icon set at this url:

https://icon-sets.iconify.design/bpmn/

But if I try to search for bmpn... nothing..

<img width="669" alt="Screenshot 2023-06-16 at 22 11 07@2x" src="https://github.com/iconify/icon-sets/assets/6997343/b4e8f3c3-a151-45a4-b39e-09153c94fa4f">

**Comment by cyberalien** [comment]
Search shows icons, not icon sets.

To find icon sets, use filter field in icon sets list. It is below search field.

**Comment by cyberalien** [comment]
Actually, it won't show up in filter either and that's a bug. That icon set is marked as hidden because it was abandoned for many years. Filter should show hidden icon sets when typing its full name or prefix, but it doesn't.

Looking at repository, it was updated recently, so it is no longer abandoned. I'll change it in update script.

**Comment by cyberalien** [comment]
Sorry, but that icon set is not coming back.

After I unmarked that set as hidden and ran parser to make sure everything works, parser ran into multiple issues and I remembered why that icon set was marked as hidden.

Icon quality is absolutely horrible. Some icons contain raster images, some use white color for parts that are supposed to be transparent, code contains massive amount of junk (though last point is not a serious issue as parser can clean it up, but it adds up to lack of quality control). Icons are designed without any consistency with different sizes, different styles, no logical grid.

That icon set was one of first sets to be added to Iconify back when I was just experimenting, there weren't many good sets available back then and pretty much all sets were designed as icon fonts. In modern web such low quality icon fonts have no place.

Bug in filter will be fixed though.
